packages/navigation-menu/README.md

```diff
@@ -0,0 +1,23 @@
+# @acpaas-ui/ngx-components/navigation-menu
+
+**NavigationMenu**
+
+This component provides a generic navigation menu that can be used in your projects, providing an administrative-like interface look and feel. The navigation menu features two basic modes: (mode selection is done automatically and cannot be configured manually)
+
+- **Mobile mode**: On mobile, the component will feature a navigation bar that places itself on the bottom of the screen. Navigation links will appear as tabs, making it possible to display a maximum of 3 tabs. If more navigation links are configured, a 'More' tab will automatically be created, embedded the invisible links under that tab. An animation highlights the navigation as the user scrolls though the different menus.
+
+- **Desktop mode**: On desktop, the component features a navigation bar that places itself on the outright left border of the screen. The desktop mode integrates two layouts: docked and undocked. The default mode can be set through module configuration. When docked, the menu only displays square icons io full navigation links. Clicking on the link will trigger opening up sub menu items in overlay mode. When not docked, the menu features different accordeons, containing the configured navigation links.
+
+## Usage
```

Comment:
Should contain an API and an examples part (or link to example) that demonstrates how to use this component. People will get stuck with how to implement this otherwise.

Reply:
I don't see many value in copying the documentation available in the styleguide in this md file (escpecially since there is a link in the md pointing to the documentation of the styleguide, where examples and code snippets are provided)